web/src/components/SaveMessage.test.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { shallow } from "enzyme";
+import moment from "moment";
+import SaveMessage from "./SaveMessage";
+
+describe("<SaveMessage />", () => {
+  let lastSaved;
+  let subject;
+
+  describe('when saved less than 1 minute ago, it displays "Saved"', () => {
+    [
+      ["1 second ago", 1],
+      ["2 seconds ago", 2],
+      ["30 seconds ago", 30],
+      ["59 seconds", 59],
+    ].forEach(([testName, seconds]) => {
+      test(testName, () => {
+        lastSaved = moment().subtract(seconds, "seconds");
+        subject = shallow(
```

Comment:
```suggestion
        const lastSaved = moment().subtract(seconds, "seconds");
        const subject = shallow(
```

This is a stylistic choice, so again something for you and @pphillips-fearless to decide how you want to do it. I'm just imagining some future version of you or another dev looking at this and wondering "where did `lastSaved` come from?" and then having to search for it. 🤷‍♂

Reply:
I like using const whenever possible

Reply:
As the file grows, it could be difficult to find these variable declarations. Your argument makes sense. 👍